src/main/java/CodeBuildCredentials.java

```diff
@@ -60,18 +60,20 @@
     private static final int ERROR_MESSAGE_MAX_LENGTH = 178;
 
     @Getter @Setter private final String accessKey;
+    @Getter @Setter private final String sessionToken;
```

Comment:
I don't think we should add a `sessionToken` attribute to CodeBuildCredentials, since the `sessionToken` is temporary